appium/webdriver/webdriver.py

```diff
@@ -345,6 +345,50 @@ def set_value(self, element: MobileWebElement, value: str) -> T:
         self.execute(Command.SET_IMMEDIATE_VALUE, data)
         return self
 
+    def flick_element(self, element: Optional[MobileWebElement] = None,
+                      xspeed: int = 0, yspeed: int = 0,
+                      xoffset: int = 0, yoffset: int = 0,
+                      speed: int = 0) -> T:
+        """Flick the element
+
+        Args:
+            element: the element to be flicked
```

Comment:
[Notes] https://github.com/appium/appium-uiautomator2-server/blob/c0d2d45740a2f9d05cf0e6c3b7b1a19e2f85d1b1/app/src/main/java/io/appium/uiautomator2/handler/Flick.java#L52

Reply:
these shortcuts were deliberately removed from java client as they are not cross-platform. 
I would rather prefer splitting platform-specific actions and assigning them to their appropriate driver classes

Reply:
thanks, I haven't taken care of `not cross-platform` (uia2).